legacy_app.py

```
@@ -87,13 +86,11 @@
 from app.services.food_store import get_food
 from app.services.intervention_trigger_engine import build_targets_next_action
 
-# tegacy BMI helpers removed from request-path (PR-457=A)
-# /plan now delegates to canonical BMI engine via compat layer
-from decimal import Decimal
-
-from core.bmi.compat_plan import legacy_plan_category
-from core.bmi.engine import _normalize_bool_flag
-from bmi_visualization import MATPLOTLIB_AVAILABLE, generate_bmi_visualization
+from app.services.bmi_compat import (
```

Flake8 (7.3.0)

[error] 'app.services.bmi_compat.MATPLOTLIB_AVAILABLE' imported but unused

(F401)